### DISCUSSION TITLE
Organizational feature for media assets

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#119] [Feedback]
In the media assets having any organizational feature like folders, tags, labels will help in reviewing , modifying pictures later.

can also have the gallery auto populated once label/tag for that cause is added.

**Comment by an-nadaa** [comment]
This is indeed a needed feature thank fully there is already an issue discussing this [here](https://github.com/strapi/strapi/issues/8612) which is already in progress [here](https://feedback.strapi.io/customization/p/media-folders). Btw to see what's comming next in the roadmap of Strapi you can see it [here](https://feedback.strapi.io/)
